docs/accelerometer.rst

```diff
@@ -58,6 +58,10 @@ Functions
 
     Return a tuple of the gesture history. The most recent is listed last.
     Also clears the gesture history before returning.
+    
+.. note::
+
+    Gestures are not updated in the background so there needs to be constant calls to some accelerometer method to do the gesture detection. Usually gestures can be detected using a loop with a small `sleep() <https://microbit-micropython.readthedocs.io/en/latest/microbit.html?highlight=sleep>`_ delay.
```

Comment:
Could we change the link to be an internal reference to https://microbit-micropython.readthedocs.io/en/latest/microbit.html?#microbit.sleep ?

http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#internal-links